src/lib/openapi/spec/pat-schema.ts

```diff
@@ -3,27 +3,42 @@ import { FromSchema } from 'json-schema-to-ts';
 export const patSchema = {
     $id: '#/components/schemas/patSchema',
     type: 'object',
+    description:
+        'An overview of a [Personal Access Token](https://docs.getunleash.io/how-to/how-to-create-personal-access-tokens).',
     properties: {
         id: {
-            type: 'number',
+            type: 'integer',
+            description:
+                'The unique identification number for this Personal Access Token.',
+            example: 1,
+            minimum: 1,
         },
         secret: {
             type: 'string',
+            description: 'The token used for authentication.',
+            example: 'user:xyzrandomstring',
         },
         expiresAt: {
             type: 'string',
             format: 'date-time',
             nullable: true,
+            description: `The token's expiration date.`,
+            example: '2023-04-19T08:15:14.000Z',
         },
         createdAt: {
             type: 'string',
             format: 'date-time',
             nullable: true,
+            example: '2023-04-19T08:15:14.000Z',
+            description: 'When the token was created.',
         },
         seenAt: {
             type: 'string',
             format: 'date-time',
             nullable: true,
```

Comment:
These three properties (`expiresAt`, `createdAt`, `seenAt`) are marked as nullable. in what cases are they null? Especially interesting is `createdAt`. surely that shouldn't ever be null, right?

If this is because this schema still describes both read and write models, then that should be clearly marked. Any properties that are optional (or ignored) no write models should call it out in their description.

Reply:
Of the 3, only `seenAt` is actually nullable, and all three are ignored on write. I've updated the description to reflect this